resources/destination-types/logzio.yaml

```
@@ -0,0 +1,122 @@
+apiVersion: bindplane.observiq.com/v1beta
+kind: DestinationType
+metadata:
+  name: logzio
+  displayName: Logz.io
+  icon: /icons/destinations/logzio.svg
+spec:
+  parameters:
+    - name: enable_logs
+      label: Enable Logs
+      description: Enable to send logs to Logz.io.
+      type: bool
+      default: true
+
+    - name: enable_metrics
+      label: Enable Metrics
+      description: Enable to send metrics to Logz.io.
+      type: bool
+      default: true
+
+    - name: enable_tracing
+      label: Enable Tracing
+      description: Enable to send spans to Logz.io.
+      type: bool
+      default: true
+
+    - name: logs_token
+      label: Logs_token
```

Label names can have spaces
```suggestion
      label: Logs Token
```